micro_ros_diagnostic_bridge/include/micro_ros_diagnostic_bridge/micro_ros_diagnostic_bridge.hpp

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2021 - for information on the respective copyright owner
+// see the NOTICE file and/or the repository https://github.com/microros/system_modes
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#pragma once
+
+#include <utility>
+#include <tuple>
+#include <string>
+#include <map>
+#include <memory>
+
+#include "rclcpp/rclcpp.hpp"
+
+#include "diagnostic_msgs/msg/diagnostic_status.hpp"
+#include "micro_ros_diagnostic_msgs/msg/micro_ros_diagnostic_status.hpp"
+
+
+static const char UROS_DIAGNOSTICS_BRIDGE_TOPIC_IN[] = "/diagnostics_uros";
+static const char UROS_DIAGNOSTICS_BRIDGE_TOPIC_OUT[] = "/diagnostics";
+
+namespace uros_diagnostic_msg = micro_ros_diagnostic_msgs::msg;
+namespace diagnostic_msg = diagnostic_msgs::msg;
+
+typedef std::pair<unsigned int, unsigned int> DiagnosticKey;
```

Comment:
I propose to introduce a tiny struct here. This allows for self-explaining member names instead of first and second. Furthermore, I would use precise-width integer types from cstdint - in particular if the values come from ROS messages, which always use precise-width integer types.